systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMakerST.java

```diff
@@ -113,10 +116,21 @@ void testMirrorMaker(ExtensionContext extensionContext) {
             .endSpec()
             .build());
 
-        verifyLabelsOnPods(testStorage.getNamespaceName(), testStorage.getClusterName(), "mirror-maker", KafkaMirrorMaker.RESOURCE_KIND);
+        final String mirrorMakerComponentName = testStorage.getClusterName() + "-mirror-maker";
 
-        verifyLabelsForConfigMaps(testStorage.getNamespaceName(), testStorage.getSourceClusterName(), null, testStorage.getTargetClusterName());
-        verifyLabelsForServiceAccounts(testStorage.getNamespaceName(), testStorage.getSourceClusterName(), null);
+        Map<String, String> mirrorMakerMatchLabel = new HashMap<>();
+        mirrorMakerMatchLabel.put(Labels.STRIMZI_CLUSTER_LABEL, testStorage.getClusterName());
+        mirrorMakerMatchLabel.put(Labels.STRIMZI_KIND_LABEL, KafkaMirrorMaker.RESOURCE_KIND);
+        mirrorMakerMatchLabel.put(Labels.STRIMZI_NAME_LABEL, mirrorMakerComponentName);
+
+        LabelSelector mirrorMakerLabelSelector = new LabelSelectorBuilder()
+                .withMatchLabels(mirrorMakerMatchLabel)
+                .build();
```

Comment:
Would it make sense to add the label selector to `KafkaMirrorMakerResource`?

Reply:
its only this one instance on this depric. mirrormaker, but i'd like to do that, is it beneficial overall?

Reply:
<img width="1239" alt="image" src="https://github.com/strimzi/strimzi-kafka-operator/assets/53821852/59499c30-e5f9-4eb5-a89d-3c9c2cc1c1a5">

you can use this in all of these methods

Reply:
just because it wasn't done before, it doesn't mean it will not be used in the future.
And even it's deprecated resource, having it in one place makes it easier during deletion process.